stages/org.osbuild.grub2

```diff
@@ -193,7 +193,7 @@ class GrubConfig:
         )
         return {terminal: val}
 
-    def write(self, tree):
+    def write(self, tree, remove_unset):
```

Comment:
Maybe rename to `bug_issue123_compat` or `force_buggy_unset` or something to make it clear from the argument that this is something one usually does not want? Also maybe a comment on top `# bus_issue123 is just there for bug compat, see issue 123`

Reply:
Well, it's the opposite (`True` is the correct behaviour), but I get your point.

Reply:
Amended.  Thanks.